caikit/runtime/http_server.py

```diff
@@ -111,6 +113,13 @@ def __init__(self, tls_config_override: Optional[aconfig.Config] = None):
 
         self.app = FastAPI()
 
+        @self.app.exception_handler(ResponseValidationError)
```

Comment:
I can't find much documentation on `ResponseValidationError`, but is this validating the handler's response rather than the request? If so, I agree that 500 is the right return value

Reply:
[This](https://fastapi.tiangolo.com/tutorial/handling-errors/#use-the-requestvalidationerror-body) is the only documentation I found which is for `RequestValidationError`, I extrapolated for `ResponseValidationError`